portal-web/test/functional/com/liferay/portalweb/macros/headless/headlessadminuser/MakeARelationshipWithSystemObjectThroughAPI.testcase

```diff
@@ -0,0 +1,320 @@
+@component-name = "portal-headless"
+definition {
+
+	property custom.properties = "feature.flag.LPS-162964=true${line.separator}feature.flag.LPS-153324=true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Object";
+	property testray.main.component.name = "Headless";
+
+	setUp {
+		TestCase.setUpPortalInstanceNoSelenium();
+
+		task ("Given a Foundation object") {
+			var objectDefinitionId2 = ObjectDefinitionAPI.createAndPublishObjectDefinition(
```

Comment:
how many object definitions do we need? Two custom object defitions and one system User, right? I'm curious why we use Id2 and Id4 here